packages/ncurses.rb

```diff
@@ -3,23 +3,23 @@
 class Ncurses < Package
   description 'The ncurses (new curses) library is a free software emulation of curses in System V Release 4.0 (SVr4), and more. — Wide character'
   homepage 'https://www.gnu.org/software/ncurses/'
-  version '6.4-20230918'
+  version 'v6_4_20231021'
   license 'MIT'
   compatibility 'all'
   source_url 'https://github.com/ThomasDickey/ncurses-snapshots.git'
-  git_hashtag 'v6_4_20230918'
+  git_hashtag 'v6_4_20231021'
```

Comment:
How about?
```
version '6_4_20231021'
git_hashtag "v#{version}"
```
Versions really shouldn't have the leading `v`.

Reply:
done